lib/routes/samd/news.ts

```diff
@@ -0,0 +1,81 @@
+import { Route, DataItem } from '@/types';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+import { load } from 'cheerio';
+import { ofetch } from 'ofetch';
+
+const dict = {
+    industry: { id: 434, name: '行业资讯' },
+    dynamic: { id: 436, name: '协会动态' },
+    notices: { id: 438, name: '重要通知' },
+    policies: { id: 440, name: '政策法规' },
+};
+
+export const route: Route = {
+    path: 'news/:type',
+    categories: ['government'],
+    example: '/samd/news/policies',
+    parameters: { type: '文章类型' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    description: `| 行业资讯 | 协会动态 | 重要通知 | 政策法规 |
+    | -------- | ------- | ------- | -------- |
+    | industry | dynamic | notices | policies |`,
```

Comment:
```suggestion
| -------- | ------- | ------- | -------- |
| industry | dynamic | notices | policies |`,
```
Indent with either 2 or 0 spaces. Indenting with 4 spaces will break the markdown rendering.